modules/react-components/src/components/message/message-with-icon.tsx

```diff
@@ -20,19 +20,19 @@ import { IdentifiableComponentInterface, TestableComponentInterface } from "@wso
 import classNames from "classnames";
 import React, { FunctionComponent, ReactElement } from "react";
 import {
-    Divider,
     Header,
     Icon,
     Message,
     MessageHeaderProps,
+    MessageProps,
     SemanticShorthandContent,
     SemanticShorthandItem
 } from "semantic-ui-react";
 
 /**
  * Proptypes for the message with icon component.
  */
-export interface MessageWithIconProps extends IdentifiableComponentInterface, TestableComponentInterface {
+export interface MessageWithIconProps extends MessageProps, IdentifiableComponentInterface, TestableComponentInterface {
```

Comment:
Can't we name this component as `Message`?

It'll infer icons from the type by default.
If needed we can add an `icon` prop to accept a ReactElement or a boolean so someone can even pass in a custom icon or remove the icon. 

If we do this we can add [a import restriction ](https://github.com/wso2/identity-apps/blob/master/.eslintrc.js#L176)in ESLint so devs will have to use the `Message` component from `react-components` instead of directly getting from `semantic`.



We can add alias to fix the import conflicts.

```
import type { MessageProps as SemanticMessageProps } from "semantic-ui-react";
import { Message as SemanticMessage } from "semantic-ui-react";



```